GDJS/Runtime/howler-sound-manager/howler-sound-manager.ts

```diff
@@ -560,42 +602,45 @@ namespace gdjs {
     ) {
       if (this._sounds[channel]) this._sounds[channel].stop();
 
-      var sound = this.createHowlerSound(
+      const sound = this.createHowlerSound(
         soundName,
-        /* isMusic= */ false
-      ).play();
-      this._sounds[channel] = sound;
-
+        /* isMusic= */ false,
+        volume / 100,
+        loop,
+        pitch
+      );
       sound.once('play', () => {
-        sound
-          .setLoop(loop)
-          .setVolume(volume / 100)
-          .setRate(pitch);
         if (this._paused) {
           sound.pause();
           this._pausedSounds.push(sound);
         }
       });
+      // We need to first play the sound in order to change its parameters like volume.
+      sound.play();
+      this._sounds[channel] = sound;
     }
 
     getSoundOnChannel(channel: integer): HowlerSound {
       return this._sounds[channel];
     }
 
     playMusic(soundName: string, loop: boolean, volume: float, pitch: float) {
-      var music = this.createHowlerSound(soundName, /* isMusic= */ true);
-      this._storeSoundInArray(this._freeMusics, music).play();
-
+      const music = this.createHowlerSound(
+        soundName,
+        /* isMusic= */ true,
+        volume / 100,
+        loop,
+        pitch
+      );
       music.once('play', () => {
-        music
-          .setLoop(loop)
-          .setVolume(volume / 100)
-          .setRate(pitch);
         if (this._paused) {
           music.pause();
           this._pausedSounds.push(music);
         }
       });
+      // We need to first play the music in order to change its parameters like volume.
```

Comment:
Are these comments useful? I don't see why we're telling this because we're not changing any parameter?

Reply:
nope, they're outdated